web/src/OverviewTable.tsx

```diff
@@ -0,0 +1,482 @@
+                status={row.values.statusLine.buildStatus}
+                lastBuildDur={row.values.statusLine.lastBuildDur}
+                alertCount={row.values.statusLine.buildAlertCount}
+                isBuild={true}
+              />
+              <OverviewTableStatus
+                status={row.values.statusLine.runtimeStatus}
+                alertCount={row.values.statusLine.runtimeAlertCount}
+              />
+            </>
+          )
+        },
+      },
+      {
+        Header: "Pod ID",
+        accessor: "podId",
+        width: "50px",
+        Cell: ({ row }: CellProps<RowValues>) => {
+          let [showCopySuccess, setShowCopySuccess] = useState(false)
+
+          let copyClick = () => {
+            copyTextToClipboard(row.values.podId, () => {
+              setShowCopySuccess(true)
+
+              setTimeout(() => {
+                setShowCopySuccess(false)
+              }, 3000)
+            })
+          }
+
+          let icon = showCopySuccess ? (
+            <CheckmarkSvg width="15" height="15" />
+          ) : (
+            <CopySvg width="15" height="15" />
+          )
+
+          function selectPodIdInput(podId: string | null) {
+            const input = document.getElementById(
+              `pod-${row.values.podId}`
+            ) as HTMLInputElement
+            input && input.select()
+          }
+
+          if (!row.values.podId) return null
+          return (
+            <PodId>
+              <PodIdInput
+                id={`pod-${row.values.podId}`}
+                value={row.values.podId}
+                readOnly={true}
+                onClick={() => selectPodIdInput(row.values.podId)}
+              />
+              <PodIdCopy
+                onClick={copyClick}
+                analyticsName="ui.web.overview.copyPodID"
+              >
+                {icon}
+              </PodIdCopy>
+            </PodId>
+          )
+        },
+      },
+      {
+        Header: "Endpoints",
+        accessor: "endpoints",
+        sortType: "basic",
+        Cell: ({ row }: CellProps<RowValues>) => {
+          // @ts-ignore
+          let endpoints = row.values.endpoints.map((ep) => {
+            return (
+              <Endpoint
+                onClick={() =>
+                  void incr("ui.web.endpoint", { action: "click" })
+                }
+                href={ep.url}
+                // We use ep.url as the target, so that clicking the link re-uses the tab.
+                target={ep.url}
+                key={ep.url}
+              >
+                <StyledLinkSvg />
+                <DetailText>{ep.name || displayURL(ep)}</DetailText>
+              </Endpoint>
+            )
+          })
+          return <div>{endpoints}</div>
+        },
+      },
+      {
+        Header: "Trigger Mode",
+        accessor: "triggerMode",
+        width: "50px",
+        Cell: ({ row }: CellProps<RowValues>) => {
+          return (
+            <TableTriggerModeToggle
```

Comment:
Should the trigger mode toggle render conditionally? (This may be a lack of knowledge of my end.) When I test locally, I see toggles showing up for all resources, including the Tiltfile, and not all of the toggles successfully toggle when I click on them.